.github/workflows/clitool-ci.yml

```diff
@@ -27,7 +34,7 @@ jobs:
       run: dotnet restore '${{ env.PROJPATH }}/Kmd.Logic.Gateway.Automation.Tool.csproj'
 
     - name: Build - PreRelease
-      run: dotnet build '${{ env.PROJPATH }}/Kmd.Logic.Gateway.Automation.Tool.csproj' --configuration Debug --no-restore -p:VersionSuffix=dev-$GITHUB_RUN_NUMBER
+      run: dotnet build '${{ env.PROJPATH }}/Kmd.Logic.Gateway.Automation.Tool.csproj' --configuration Debug --no-restore -p:VersionSuffix=dev-${{ steps.current-time.outputs.formattedTime }}
```

Comment:
Any reason why we have changed build number to date and time

Reply:
Sorting packages by the NuGet manager does not support NaturalSort. I mean, that when we have *-92 and *-117 versions, then *-92 is considered as newer than *-117, because they are firstly sorted by first characters (9 and 1). So I decided to put here the date, as it is sorting reliable.

More here: https://en.wikipedia.org/wiki/Natural_sort_order